charts/teamcity/values.yaml

```diff
@@ -86,6 +86,15 @@ securityContext: {}
 #  runAsNonRoot: true
 #  runAsUser: 1000
 
+volumePermissions: {}
+#  enabled: true
+#  image:
+#    image: busybox
+#    imageTag: latest
+#    pullPolicy: IfNotPresent
+#  securityContext:
+#    runAsUser: 0
+
```

Comment:
no need to manage it so precise
kind of simple `fixPermissions: true/false` should be enough